presto-hive/pom.xml

```diff
@@ -17,6 +17,12 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-parser</artifactId>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
Move this next to other test-scoped dependencies in `<!-- for testing -->` section.